allennlp/models/ensemble/ensemble.py

```diff
@@ -0,0 +1,62 @@
+from typing import Dict, List
+
+import torch
+
+from allennlp.common.checks import ConfigurationError
+from allennlp.common.params import Params
+from allennlp.models.model import Model
+from allennlp.models.model import remove_pretrained_embedding_params
+
+class Ensemble(Model):
+    """
+    An ensemble runs multiple instances of a model and selects an answer from the subresults via some
+    ensembling strategy.
+
+    Ensembles differ from most models in that they do not have a vocabulary or weights of their own
+    (instead they rely on the vocabulary and weights from submodels) and they are not directly
+    trainable.  Instead, the submodels are trained independently and the ensemble is created
+    from the result.
+    """
+
+    def __init__(self,
+                 submodels: List[Model]) -> None:
+        vocab = submodels[0].vocab
+        for submodel in submodels:
+            if submodel.vocab != vocab:
+                raise ConfigurationError("Vocabularies in ensemble differ")
+
+        super(Ensemble, self).__init__(vocab, None)
+
+        # Using ModuleList propagates calls to .eval() so dropout is disabled on the submodels in evaluation
+        # and prediction.
+        self.submodels = torch.nn.ModuleList(submodels)
+
+    def forward(self, *inputs) -> Dict[str, torch.Tensor]:  # pylint: disable=arguments-differ
```

Comment:
I don't think having this here adds anything.  You're just overriding an abstract method with another abstract method, and not changing the API.

Reply:
Yeah, this class is still abstract, and pylint doesn't know that.  So just disable the `abstract-method` check underneath the class docstring.